scalableminds/webknossos
# Update team members

### Steps to test:
- Go to team view under `/teams` route. There should be an action called `Add / Remove Users`
![image](https://user-images.githubusercontent.com/57364771/233603695-2daf30b9-ae04-4238-ba16-8bb4ba305c40.png)
- Expand Team and make sure only active users are listed
### Issues:
- fixes #6958 

------
(Please delete unneeded items, merge only when none are left open)
